judo-ui-react/src/main/resources/actor/src/layout/index.tsx.hbs

```diff
@@ -1,6 +1,54 @@
 {{> fragment.header.hbs }}
 
-export * from './Footer';
-export * from './Header';
-export * from './Layout';
-export * from './Navigator';
+import { Outlet } from 'react-router-dom';
+import { useTheme } from '@mui/material/styles';
+import { useMediaQuery, Box, Container, Toolbar } from '@mui/material';
+import { CustomBreadcrumb } from '~/components/CustomBreadcrumb';
+import { useConfig } from '~/hooks';
+import { MenuOrientation } from '~/config';
+import { Drawer } from './Drawer';
+import { Header } from './Header';
+import { Footer } from './Footer';
+import { HorizontalBar } from './Drawer/HorizontalBar';
+
+export const Layout = () => {
+  const theme = useTheme();
+  const matchDownXL = useMediaQuery(theme.breakpoints.down('xl'));
+  const downLG = useMediaQuery(theme.breakpoints.down('lg'));
+
+  const { container, miniDrawer, menuOrientation, onChangeMiniDrawer } = useConfig();
+
+  const isHorizontal = menuOrientation === MenuOrientation.HORIZONTAL && !downLG;
+
+  // set media wise responsive drawer (auto-close at start based on width)
+  // useEffect(() => {
+  //   if (!miniDrawer) {
+  //     onChangeMiniDrawer(!matchDownXL);
+  //   }
+  // }, [matchDownXL]);
```

Comment:
comment?

Reply:
As you can see, the behavior is explained in the comment. I'm certain we don't want this, not sure the original template why did it. We are keeping this one as well in case we can sell this later as a feature.